### ISSUE TITLE
Update normalize.css and have a better strategy for including normalize

### ISSUE THREAD

**Issue by sambhav-gore** [zalando/dress-code#376]
Update to latest version of Normalize https://github.com/necolas/normalize.css 

We can also think about a better strategy for including the normalize lib.

**Comment by rbarilani** [comment]
@sambhav-gore What if we don't use it at all? [this answer](http://getbem.com/faq/#global-css-resets) convinced me

**Comment by sambhav-gore** [comment]
@rbarilani Yes, that is a better suggestion, I would prefer each block resetting itself over including normalize.
